File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L5/04/Kasus4.ts
/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
function konversi(miles: number): number {
    return miles * 1.609;
}

function main(): void {
    console.log("Miles\t\tKilometers");
    console.log("-------------------------------");
    // Use for loop
    for (let miles = 1; miles <= 10; miles++) {
        console.log(`${miles}\t\t${konversi(miles)}`);
    }
}

main();

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-04/plagiarized/L6/04/Kasus4.kt
package IR

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
object Kasus4 {
    fun Konversi(miles: Int): Double {
        var miles = miles
        if (miles <= 10) {
            println(miles.toString() + "\t\t" + miles * 1.609)
            return Konversi(++miles)
        }
        return 0
    }

    @JvmStatic
    fun main(args: Array<String>) {
        println("Miles\t\tKilometers")
        println("-------------------------------")
        // Use while loop
        Konversi(1)
    }
}
